### PR TITLE
Document database tradeoffs in README.md

### PR DESCRIPTION
**Pull request by chrismytton**
This may not be the best place for this, but I wanted to get my thoughts down.

### CODE REVIEW

**Comment by SimKennedy** [comment]
@chrismytton Thanks for this. It's a good summary and great to hear your perspective. I chose the postgres view approach to make it easy to change the data structure while in Alpha, which we have done a few times in light of other problems. If/when a more relational schema is needed it shouldn't be too difficult to turn the Views into Tables and use ForeignKeys where needed.

**Comment by chrismytton** [comment]
Think this is ready for a proper review now.

File: README.md [hunk 1]
```diff
@@ -215,30 +215,62 @@ To run use this command
...
+The Postgres views are embedded and managed from the models using the python package `django-pgviews-redux`.
+
+https://pypi.org/project/django-pgviews-redux/
 
 Pros
 
 - Simple to load the data in, just add JSON
 - Don't have to define the schema up-front
 - Can do ad-hoc queries on the full data
+- Easy to update the models during development
```
Comment (chrismytton) [inline README.md:230]: Yeah this has been really handy actually!

File: README.md [hunk 2]
```diff
@@ -215,30 +215,62 @@ To run use this command
...
+Linking the datasets proved tricky with the various identifier schemas in each dataset and duplicated identifiers
+in the BODS data.
+
+https://standard.open-contracting.org/latest/en/schema/identifiers/#identifiers
+http://standard.openownership.org/en/0.2.0/schema/guidance/identifiers.html
+
+A workaround was used to build "helper" functions to abstract the queries needed to retrieve linked
+data without needing to hardcode the relationships into the models using ForeignKey or ManyToMany fields. 
+
+Other Alternatives
```
Comment (chrismytton) [inline README.md:265]: Could make this a level 2 heading?

**Pull request merged**